spikeinterface/toolkit/postprocessing/correlograms.py

```diff
@@ -3,8 +3,8 @@
 
 
 def compute_correlograms(sorting, 
-                    window_ms=100.0, bin_ms=1.0,
-                    symmetrize=False):
+                         window_ms=100.0, bin_ms=1.0,
+                         sortsymmetrize=False):
```

Comment:
With using `sortsymmetrize` instead of `symmetrize`

Reply:
That's what used in the code

Reply:
??? Maybe a mistake I think. Lets go back to symmetrize

Reply:
fixed